src/test/java/ResponseHandlerTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class ResponseHandlerTest {
+    @Test
+    public void outputStream_writeResponseMessage() throws IOException {
+        ResponseHandler responseHandler = new ResponseHandler();
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
+        String responseMessage = "hello world";
+        responseHandler.prepare(output, responseMessage);
```

Comment:
What would happen if you made the `prepare` method take a `byte array` as standard, instead of a stream?